accumulo/src/test/java/org/vertexium/accumulo/AccumuloResource.java

```diff
@@ -0,0 +1,148 @@
+package org.vertexium.accumulo;
+
+import org.apache.accumulo.core.client.AccumuloException;
+import org.apache.accumulo.core.client.AccumuloSecurityException;
+import org.apache.accumulo.core.client.Connector;
+import org.apache.accumulo.minicluster.MiniAccumuloCluster;
+import org.apache.accumulo.minicluster.MiniAccumuloConfig;
+import org.junit.rules.ExternalResource;
+import org.vertexium.GraphConfiguration;
+import org.vertexium.test.GraphTestBase;
+import org.vertexium.util.VertexiumLogger;
+import org.vertexium.util.VertexiumLoggerFactory;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+
+import static org.vertexium.test.GraphTestBase.*;
+
+public class AccumuloResource extends ExternalResource {
```

Comment:
@joeferner @mwizeman I did split the GraphTestBase into multiple tests but I did not push it through yet because it was kind of ugly and I was hoping to find a better way. I'll try to summarize. I split GraphTestBase into multiple "suites" that could be called from "GraphTestBase ". That works fine but the "suites" are meant to be independent. In order to share a common environment, you have to make the suites refer to a "static" environment that has the graph, authorizations, etc. However, those members are defined at instantiation time (i.e. different for Accumulo, InMemory, etc). Since you cant override static functions in Java (for a valid philosophical reason), I kinda had to make an Common Environment class that gets instantiated by the GraphTest subclasses but overrides a static variable than the suites can access to. It works but I dont feel too happy about overriding static member once a class has been instantiated (seems wrong). I can push my changes to my branch (they are on my machine at home) and let you guys look at it to see what I am talking about. I couldnt think of a better way to split GraphTests without major refactoring.